tests/jsTests/getHelpNowSpec.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Suite : Tests for show get help action section
+ * Description : This is a test suite for checking if the get-help-action section (call/sms) is visible
+ */
+describe("Tests for show get help action section", function() {
+	beforeEach(function() {
+		jasmine.clock().install();
+		setFixtures('<div id="get-help-section"></div> \
+				<div id="get-help-action-section"></div>');
+		spyOn($.fn, "fadeOut").and.callFake(function() {
```

Comment:
These are mock function calls for supporting jquery's fadeOut and fadeIn. Technically, fadeOut hides and fadeIn shows the element with a delay. I have mocked these function calls to test these functionalities. show and hide functions are functions of jquery.

Reference: [Intro to jasmine](https://jasmine.github.io/2.0/introduction.html) | [hide](http://api.jquery.com/hide/) | [show](http://api.jquery.com/show/)